### ISSUE TITLE
Ending Splashscreen enhancement

### ISSUE THREAD

**Issue by CoLoRaptor** [colobot/colobot#472]
Some informations need to be added and modified. I think it should be like this:

![schowek01](https://cloud.githubusercontent.com/assets/4019345/7569077/c04b20a8-f806-11e4-9d78-5bd1f2277304.png)

**Comment by OdyX** [comment]
By the way, (being on that screen), I think the initial developers (and betatesters) list could probably be dropped in favour of something more relevant to the actual team.

Beware with logos too: they are most probably not open-source.

**Comment by krzys-h** [comment]
Colobot logo was released as part of the data package that was licensed under GPLv3 along with the code.

TerranovaTeam logo is our logo so we can use it without a problem - https://github.com/colobot/colobot-misc/blob/master/logos/TerranovaTeam-Logo%26Title-Vector.svg

I'm not sure about the EPSITEC one. The one that we were given in the original files is different.
![](https://raw.githubusercontent.com/colobot/colobot-data/c5926eaa422b683bf5a5b380527987258b8a7444/textures/interface/generico.png)
I remember that at some point they gave us their logo via email, but that one was black. I'm not sure how it's licensed either.
![logo epsitec](https://cloud.githubusercontent.com/assets/1517255/7703983/d64f96d2-fe3b-11e4-8aaa-6ae902672afb.png)

**Comment by tomangelo2** [comment]
I don't know how it works with logos, but if this blue logo was released on GPLv3, then IMHO if brighter version (that we're using at the moment) was made from original one, then it shouldn't be a problem.

**Comment by CoLoRaptor** [comment]
@OdyX & @krzys-h & @tomangelo2 : I made that logo blue and without "SA" part to make it look better and more compatible with mostly known logo from original Colobot game, as they used that name in 2001-2004 (All CeeBot versions also contain that). New render is brighter to fit this with present logo in intro3.png as they never explained, what exactly their color is.

![GitHub Logo](https://raw.githubusercontent.com/colobot/colobot-data/dev/textures/interface/intro3.png)

They sent us only rasterized, non-transparent PNG logo, so I vectorized it and made it blue some time ago, but didn't find time to ask if I should commit this logo to colobot-misc/master repo.

By the way, I think they probably changed company's name to Cresus, as the on their official site there's almost no EPSITEC logo and name and it's working on several domains (http://epsitec.com/ ; http://epsitec.ch/ ; http://www.cresus.ch/). I think we should for sure write to them another letter to figure it out how we should using this.
